pkg/config/ingress.go

```diff
@@ -17,3 +18,10 @@ type RouteAdmissionPolicy struct {
 	// +kubebuilder:default="InterNamespaceAllowed"
 	NamespaceOwnership NamespaceOwnershipEnum `json:"namespaceOwnership"`
 }
+
+type IngressPortsConfig struct {
+	// +kubebuilder:default=80
+	Http uint16 `json:"http"`
+	// +kubebuilder:default=443
+	Https uint16 `json:"https"`
+}
```

Comment:
Do we support disabling individual protocols? How do you tell the difference between a configuration where HTTP is explicitly disabled vs. one where the config does not mention HTTP at all?

Reply:
OpenShift router does not support disabling ports through any Ingress/IngressController APIs and we aligned to that. Routes are also involved in how they are exposed (https or http), it depends on the configuration they have.

Reply:
OK, sounds good